great_expectations/cli/datasource.py

```diff
@@ -621,19 +771,51 @@ def _load_file_as_data_asset_from_pandas_datasource(context, data_source_name):
 
             batch_kwargs["reader_method"] = reader_methods[option_selection]
 
-            batch = datasource.get_data_asset(data_asset_name, batch_kwargs=batch_kwargs)
+            batch = datasource.get_data_asset(data_asset_name,
```

Comment:
This is confusing parameter naming, referenced above; data_asset_name -> generator_asset

Reply:
+1 - will change